test/functional/test_runner.py

```diff
@@ -206,6 +206,7 @@
     'wallet_createwallet.py --descriptors',
     'wallet_watchonly.py --legacy-wallet',
     'wallet_watchonly.py --usecli --legacy-wallet',
+    'wallet_reindex.py',
```

Comment:
In bc8fb2a9d5fe941bcc77eacb23d72c663828f8e3 "test: coverage for wallet birth time interaction with -reindex"

Needs `--legacy-wallet` and `--descriptor`.

Reply:
Done.
Plus, discovered and fixed an issue with the legacy wallet birth time along the way.